File: src/main/java/com/indiepost/model/Option.java
package com.indiepost.model;

import javax.persistence.*;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;
import java.io.Serializable;

/**
 * Created by jake on 7/25/16.
 */
@Entity
@Table(name = "Options")
public class Option implements Serializable {

    private static final long serialVersionUID = -6249744799288631153L;

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @NotNull
    @Size(min = 2, max = 20)
    private String name;

    @NotNull
    @Size(min = 2, max = 20)
    private String value;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: src/main/kotlin/com/indiepost/model/Option.kt
package com.indiepost.model

import java.io.Serializable
import javax.persistence.*
import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by jake on 7/25/16.
 */
@Entity
@Table(name = "Options")
data class Option(
        @Id
        @GeneratedValue(strategy = GenerationType.AUTO)
        var id: Long? = null,

        @NotNull
        @Size(min = 2, max = 20)
        var name: String? = null,

        @NotNull
        @Size(min = 2, max = 20)
        var value: String? = null
) : Serializable {
    companion object {
        private const val serialVersionUID = -6249744799288631153L
    }
}
